src/Employee.java

```diff
@@ -1,13 +1,11 @@
-import java.util.Arrays;
 import java.util.Objects;
 
 public class Employee {
-    //String[] arrFio = new String[10];
-    //int[] arrDep = new int[10];
-    //int[] arrSalary = new int[10];
     private String fio;
-    private int dep, salary;
-    int id = 0;
+    private int dep;
+    private int salary;
+    private static int id = 0;
```

Comment:
```suggestion
    private static int idGenerator = 0;
```
лучше назвать как-то так, потому что это не id